windows/NsisInclude/FilesToUninstall.nsh

```diff
@@ -0,0 +1,612 @@
+Delete "$INSTDIR\etc\fonts\conf.d\35-repl-custom.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\37-repl-webfonts.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\38-repl-webfonts-custom.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\40-non-latin-microsoft.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\40-non-latin-misc.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\40-nonlatin.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\45-latin-microsoft.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\45-latin-misc.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\45-latin.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\49-sansserif.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\50-user.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\51-local.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\57-dejavu-sans-mono.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\57-dejavu-sans.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\57-dejavu-serif.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\60-latin.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\65-fonts-persian.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\65-nonlatin.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\67-override-aliases.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\68-override.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\69-unifont.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\80-delicious.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\82-no-embedded-bitmaps.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\82-no-force-autohint.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\82-no-ttf-as-bitmap.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\83-yes-bitmaps.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\83-yes-postscript.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\88-forced-synthetic.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\90-non-tt-fonts.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\90-synthetic.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\90-tt-fonts-microsoft.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\90-tt-fonts-misc.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\92-selective-rendering-microsoft.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\92-selective-rendering-misc.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\93-final-rendering.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\94-no-synthetic.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\95-reject.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\97-selective-rendering-custom.conf"
+Delete "$INSTDIR\etc\fonts\conf.d\README"
+RMDir "$INSTDIR\etc\fonts\conf.d"
+Delete "$INSTDIR\etc\fonts\fonts.conf"
+RMDir "$INSTDIR\etc\fonts"
+Delete "$INSTDIR\etc\dbus-1\system.conf"
+RMDir "$INSTDIR\etc\dbus-1"
+RMDir "$INSTDIR\etc"
+Delete "$INSTDIR\data\app-menu.ui"
+RMDir "$INSTDIR\data"
+Delete "$INSTDIR\bleachbit.exe"
+Delete "$INSTDIR\bleachbit_console.exe"
+Delete "$INSTDIR\cacert.pem"
+Delete "$INSTDIR\COPYING"
+Delete "$INSTDIR\gi.overrides._gi_gst.pyd"
+Delete "$INSTDIR\gi._gi.pyd"
+Delete "$INSTDIR\gi._gi_cairo.pyd"
+Delete "$INSTDIR\gspawn-win32-helper-console.exe"
+Delete "$INSTDIR\gspawn-win32-helper.exe"
+Delete "$INSTDIR\libatk-1.0-0.dll"
+Delete "$INSTDIR\libcairo-gobject-2.dll"
+Delete "$INSTDIR\libepoxy-0.dll"
+Delete "$INSTDIR\libffi-6.dll"
```

Comment:
I tested AppVeyor build 1865. The installation seemed OK. When I ran it, it seemed OK, but when I uninstalled it, there were a few files left

libffi-6.dll
libgio-2.0-0.dll
libglib-2.0-0.dll
libgmodule-2.0-0.dll
libgobject-2.0-0.dll
libintl-8.dll
libwinpthread-1.dll
libzzz.dll
msvcr100.dll

These are listed in the uninstall script. I wonder whether the files were locked, but I don't remember seeing that before. I plan to re-test this, so this note can help remind me. 

For now, what do you think about adding `/REBOOTOK` flag to the Delete commands for executables/libraries (pyd, dll, exe)?

Reply:
Uninstall is clean in build 1883 (commit) e6af6c433915b38383fda29cde4e8e70eb3e6c68